data/config/celery.toml

```diff
@@ -0,0 +1,2 @@
+event_queue_prefix = "fidesops_worker"
+default_queue_name = "fidesops"
```

Comment:
A quirk of this implementation is that Celery will add any variable specified here to the config, and not throw an error if the variable is not specified in the [config spec](https://docs.celeryq.dev/en/stable/userguide/configuration.html#new-lowercase-settings), and Celery will ignore it.